docker/cron/dump-crontab

```diff
@@ -1,10 +1,11 @@
+MAILTO=""
 
-## Trigger a full dump on 1st and 15th of every month
-00 20 1,15 * * lbdumps /code/listenbrainz/admin/create-dumps.sh full
-## Around 24 hours later, trigger a full import into the spark cluster
-00 21 2,16 * * lbdumps /usr/local/bin/python /code/listenbrainz/manage.py spark request_import_full
+## Trigger a full dump on 1st and 15th of every month, in the middle of the night, do not block to wait for lock.
+0 0 1,15 * * lbdumps flock -x -n /var/lock/lb-dumps.lock /code/listenbrainz/admin/create-dumps.sh full >> /logs/dumps.log 2>&1
```

Comment:
just to confirm that you're aware that this will only lock within the currently running container.
The issue that you found previously when cron was running in a separate container won't be fixed by this lock, as the lock files aren't visible by all containers

Reply:
Yes, that is not the problem I am trying to solve here -- these locks are for preventing cron jobs in the same container from stepping on each other.